.gitignore

```diff
@@ -4,7 +4,11 @@
 
 # Environment Specific #
 #####################
-/Projects/SeniorProject/devcycle-server/dataCollection/wsgi.py
+/dataCollection/wsgi.py
+/dataCollection/settings.py
+*.swp
+*.swo
+*~
 
 # CTAGS FILE #
-/Projects/SeniorProject/devcycle-server/dataCollection/tags
+/dataCollection/tags
```

Comment:
Oh maybe, now that you mention it I'm not sure. I was assuming it didn't because of the path it was previously.